.gitignore

```diff
@@ -0,0 +1 @@
+.vscode
```

Comment:
I think we should ignore var-option in our project

Reply:
okay